src/Xamarin.Android.Build.Tasks/Xamarin.Android.Application.targets

```diff
@@ -35,4 +35,29 @@ Copyright (C) 2019 Microsoft Corporation. All rights reserved.
     </GetAndroidPackageName>
     <Exec Command="&quot;$(AdbToolPath)adb&quot; $(AdbTarget) shell am force-stop &quot;$(_AndroidPackage)&quot;" />
   </Target>
+  <PropertyGroup>
+    <AndroidAotCustomProfile Condition=" '$(AndroidAotCustomProfile)' == '' ">custom.aprof</AndroidAotCustomProfile>
```

Comment:
This property should also be separately documented, in the Properties section of `BuildProcess.md`.

`$(AndroidAotCustomProfilePath)` (appending `Path`) might make it clearer that this needs to contain a filename.  Or perhaps `AndroidAotCustomProfileOutputPath`?

Reply:
Changed to `AndroidAotCustomProfilePath`